test/turbopack-tests-manifest.json

```diff
@@ -3367,7 +3366,8 @@
       "app dir - metadata dynamic routes text routes should not throw if client components are imported but not used"
     ],
     "failed": [
-      "app dir - metadata dynamic routes should error if the default export of dynamic image is missing"
+      "app dir - metadata dynamic routes should error if the default export of dynamic image is missing",
```

Comment:
so these test seems unexpectedly passing due to some of code paths in turbopack in a certain way regardless of the condition, and fixing those reveals another failure paths - I may need debug & fix to bring this back.